docs-en/chapter_array_and_linkedlist/linked_list.md

```diff
@@ -390,13 +390,13 @@ Building a linked list involves two steps: initializing each node object and the
     n3.next = &n4;
     ```
 
-An array is a single variable, such as the array `nums` containing elements `nums[0]`, `nums[1]`, etc., while a linked list is composed of multiple independent node objects. **We usually refer to the linked list by its head node**, as in the linked list `n0` in the above code.
+An array is one cohesive variable, for instance, the array nums includes elements like nums[0], nums[1], and so on, whereas a linked list is made up of several distinct node objects.  **We typically identify a linked list by its head node**, for example, the linked list in the previous code snippet is referred to as `n0`.
 
 ### Inserting a Node
 
-Inserting a node in a linked list is very easy. As shown in the image below, suppose we want to insert a new node `P` between two adjacent nodes `n0` and `n1`. **This requires changing only two node references (pointers)**, with a time complexity of $O(1)$.
+Inserting a node into a linked list is straightforward. As depicted in Figure 4-6, let's assume we aim to insert a new node P between two adjacent nodes n0 and n1.  **This can be achieved by simply modifying two node references (pointers)**, resulting in a time complexity of $O(1)$.
```

Comment:
“depicted" -> "shown" (try to avoid complex words when a simple one can be used).
please add "`" around P, n0, n1.
please remove the extra space leading "**This".